packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/ImageCaptureTest.java

```diff
@@ -190,12 +190,8 @@ public void takePicture_usesExpectedOnImageSavedCallback() {
         mock(ImageCapture.OutputFileResults.class);
     final String mockFileAbsolutePath = "absolute/path/to/captured/image";
     final ImageCaptureException mockException = mock(ImageCaptureException.class);
-    final int testImageCaptureError = 54;
```

Comment:
Why did these lines need to be removed for this PR? Unrelated test failure?

Reply:
A warning indicated these values aren't used. Unrelated fix!